packages/common/src/TitleBar.ts

```diff
@@ -54,16 +64,55 @@ export class Button extends HTMLWidget {
         this._enabled = _;
         return this;
     }
+
+    getBBox(refresh = false, round = false) {
+        if (refresh || this._boundingBox === null) {
+            const domNode = this._element.node() ? this._element.node() : null;
+            if (domNode instanceof Element) {
+                const rect = domNode.getBoundingClientRect();
+                this._boundingBox = {
+                    x: rect.left,
+                    y: rect.top,
+                    width: rect.width,
+                    height: rect.height
+                };
+            }
+        }
+        if (this._boundingBox === null) {
+            return {
+                x: 0,
+                y: 0,
+                width: 0,
+                height: 0
+            };
+        }
+        return {
+            x: (round ? Math.round(this._boundingBox.x) : this._boundingBox.x) * this._widgetScale,
+            y: (round ? Math.round(this._boundingBox.y) : this._boundingBox.y) * this._widgetScale,
+            width: (round ? Math.round(this._boundingBox.width) : this._boundingBox.width) * this._widgetScale,
+            height: (round ? Math.round(this._boundingBox.height) : this._boundingBox.height) * this._widgetScale
+        };
+    }
+
```

Comment:
How is this different to the default version in HTML Widget?

Reply:
The default version in HTML Widget returns the bbox of `this._element.node().firstElementChild`. That was causing problems with TitleBar.

Reply:
I saw that, but was wondering why?  (probably related to title bar not wanting to 100% high like most widgets?)

That being said, you could add a new method to the HTMLWIdget:
```
calcBBox(domNode: HTMLElement, refresh:boolean = false, round:boolean = false) {
...
}
```
And have both your variants call it?

Reply:
Sounds good